### DISCUSSION TITLE
📢 Announcement: Using CSLA 5: Blazor and WebAssembly book available

### DISCUSSION THREAD

**Discussion by rockfordlhotka** [MarimerLLC/csla#1641] [General]
Earlier today I released CSLA .NET version 5.2.0 with support for the new release of Blazor WebAssembly.

I am very happy to announce that the [CSLA 5: Blazor and WebAssembly](https://store.lhotka.net/using-csla-5-blazor-and-webassembly) book, including updates for the latest version of Blazor and of CSLA .NET is now in the store.

If you previously purchased the book, you can return to the store to download the updated content.

If you haven't yet purchased the book, now is a great time to go to the store and buy it!

**Comment by Chicagoan2016** [comment]
@rockfordlhotka , just downloaded the latest version : )

**Comment by wilfreds** [comment]
Hi, If you are a 50$ sponsor. How do you get access to the book?

**Reply by rockfordlhotka** [reply]
@wilfreds please send me (rocky at lhotka dot net) an email and I'll make sure you are set up.

One drawback with the way GitHub does the sponsorships is that I don't get any contact info for sponsors (privacy I suppose?) and so I can't proactively do much for people until they email me.
